corehq/apps/domainsync/management/commands/copy_domain.py

```diff
@@ -239,9 +243,20 @@ def run(self):
                           (doc["doc_type"], count, self.total, doc["doc_type"], doc["_id"])
                 else:
                     if not self.simulate:
-                        dt = DocumentTransform(doc, self.sourcedb, self.exclude_attachments)
-                        save(dt, self.targetdb)
+                        for i in reversed(range(5)):
+                            try:
+                                dt = DocumentTransform(doc, self.sourcedb, self.exclude_attachments)
+                                break
+                            except RequestError:
+                                if i == 0:
+                                    raise
```

Comment:
No since that the outer try block logs the error to a file so that you can retry if you need to